### PR TITLE
CASM-3871 Grafana Upgrade and Dashboard improvements

### PR DESCRIPTION
**Pull request by sumithrb**
## Summary and Scope

_Grafana Upgrade and Dashboard improvements by supporting new visualizations_
_Upgrades Grafana to 9.3.2 similar to HPCM and SMA_

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves [CASM-3871](https://jira-pro.its.hpecorp.net:8443/browse/CASM-3871)

## Testing

_List the environments in which these changes were tested._

### Tested on:

  * `frigg`
  * Local development environment
  * Virtual Shasta

### Test description:

- Was upgrade tested? YES.

Untitled

## Risks and Mitigations

_No known issues with these changes and no special considerations._

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] Testing is appropriate and complete, if applicable

### CODE REVIEW

**Review by rambabubolla — APPROVED** [review]

**Review by danielvaz46 — CHANGES_REQUESTED** [review]
The piechart panel we are adding through the zip file is deprecated in Grafana 9.3.2. It is also no longer necessary as Grafana 8.x+ comes with the panel pre-installed. Please remove it, thanks.

**Comment by sumithrb** [comment]
> The piechart panel we are adding through the zip file is deprecated in Grafana 9.3.2. It is also no longer necessary as Grafana 8.x+ comes with the panel pre-installed. Please remove it, thanks.

Removed the pie-chart zip lines. Check if the Vonage status panel still needs to be installed as done.

File: docker.io/grafana/grafana/9.3.2/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+RUN apk add --upgrade apk-tools && \
+    apk add wget zip unzip mysql-client nodejs npm git && \
+    apk update && apk -U upgrade && \
+    rm -rf /var/cache/apk/*
+
+RUN mkdir /var/lib/grafana-plugins
+
+#Vonage-status-panel
+RUN wget -nv https://grafana.com/api/plugins/vonage-status-panel/versions/latest/download -O /var/lib/grafana-plugins/vonage-status-panel-1.0.11.zip
```
Comment (danielvaz46) [inline docker.io/grafana/grafana/9.3.2/Dockerfile:36]: Are we sure that we have permission to download from the panel from the internet directly?
The usual practice is that we either download it from our artifactory or we add a zip file to the repository which we can access using the COPY command within the Dockerfile.
Other than this, looks fine.

**Review by danielvaz46 — APPROVED** [review]

**Comment by sumithrb** [comment]
> The piechart panel we are adding through the zip file is deprecated in Grafana 9.3.2. It is also no longer necessary as Grafana 8.x+ comes with the panel pre-installed. Please remove it, thanks.

@danielvaz46 I am getting the following error for piechart panels in dashboards while testing without using the old chart, I believe we still have to use the wget method for the plugin to work - 
image

File: docker.io/grafana/grafana/9.3.2/Dockerfile (hunk 1 above)
Comment (sumithrb) [reply docker.io/grafana/grafana/9.3.2/Dockerfile:36]: Yes, these are open-source panels and hence, we can download them directly.

**Review by rambabubolla — APPROVED** [review]

**Pull request merged**